https://issues.redhat.com/browse/ACM-22475 Backup and restore Observability (#8222)
open-cluster-management/rhacm-docs: business_continuity/backup_restore/backup_restore_config_obs.adoc

[#backup-restore-obs-config]
= Backup and restore configuration for Observability

> The Observability service uses an S3-compatible object store to persist all time-series data collected from managed clusters. Because Observability is a stateful service, it is sensitive to active and passive backup patterns. You must configure Oservability to ensure data continuity and integrity during hub cluster migration or backup.

Reviewer: "persist" is a bit of an uncommon verb choice. might not be good for translation. do you think you could use "maintain, preserve, keep"? 

```suggestion
The Observability service uses an S3-compatible object store to keep all time-series data collected from managed clusters. Because Observability is a stateful service, it is sensitive to active and passive backup patterns. You must configure Oservability to ensure data continuity and integrity during hub cluster migration or backup.
```

Author: I like your suggestion!

> The Observability service uses an S3-compatible object store to persist all time-series data collected from managed clusters. Because Observability is a stateful service, it is sensitive to active and passive backup patterns. You must configure Oservability to ensure data continuity and integrity during hub cluster migration or backup.

Reviewer: similar to my other comment. I'm not used to seeing phrasing like "data continuity and integrity" what do you think of this rewrite? 
```suggestion
The Observability service uses an S3-compatible object store to persist all time-series data collected from managed clusters. Because Observability is a stateful service, it is sensitive to active and passive backup patterns. You must configure Oservability to ensure that your data stays safe and keeps its continuity during the hub cluster migration or backup.
```

*Notes:* 

> - When a managed cluster is detached from the primary hub cluster and reattached to the backup hub cluster, metrics are not collected. To minimize gaps, consider scripting the cluster migration for large fleets.

Reviewer: Although "consider" is perfectly fine to use, what do you think about us being more assertive? Also, "minimize" I think is a flagged word in the Vale style guide. What do you think about rephrasing so it's more metrics focused? 

```suggestion
- When a managed cluster is detached from the primary hub cluster and reattached to the backup hub cluster, metrics are not collected. To help connect the metrics, you can script the cluster migration for large fleets.
```

- For product backup and restore, the Observability service automatically labels its resources with the `cluster.open-cluster-management.io/backup` label.

> .Resources that are automatically backuped up and restored for Observability

Reviewer: Awesome table. Renders really nicely :)

|====
| Resource type | Resource name  

| ConfigMaps 
| `observability-metrics-custom-allowlist`, `thanos-ruler-custom-rules`, `alertmanager-config`, `policy-acs-central-status`, Any ConfigMap labeled with `grafana-custom-dashboard`

| Secrets
| `thanos-object-storage`, `observability-server-ca-certs`, `observability-client-ca-certs`, `observability-server-certs`, `observability-grafana-certs`, `alertmanager-byo-ca`, `alertmanager-byo-cert`, `proxy-byo-ca`, `proxy-byo-cert`
|====

== Additional resources

- For the steps to complete the backup and restore for Observability, see xref:../backup_restore/backup_restore_obs.adoc#backup-restore-obs[Backing up and restoring Observability service].

> 

Reviewer 2: This topic looks good.

[#backup-restore-obs-config]
= Backup and restore configuration for Observability

The Observability service uses an S3-compatible object store to persist all time-series data collected from managed clusters. Because Observability is a stateful service, it is sensitive to active and passive backup patterns. You must configure Oservability to ensure data continuity and integrity during hub cluster migration or backup.

*Notes:* 

- No metrics are collected from a managed cluster between the time it is detached from the primary hub and re-attached to the backup hub. To minimize gaps, consider scripting the cluster migration for large fleets.

- For product backup and restore, the Observability service automatically labels its resources with the `cluster.open-cluster-management.io/backup` label.

.Prerequisites

> - Use the local-cluster renaming feature to assign unique names to each hub cluster. This helps disambiguate metrics for each hub cluster in Grafana.

Reviewer 2: or namespace?

Reviewer 2: This may be a prereq list that needs to actually go in the procedure.

> - Use the local-cluster renaming feature to assign unique names to each hub cluster. This helps disambiguate metrics for each hub cluster in Grafana.

Author: ```suggestion
```

> - Enable self-management on your hub cluster to ensures its metrics are collected and visible in Grafana. The Observability service recognizes the hub cluster as a managed cluster.

Reviewer 2: need to look at the local cluster feature in adv config.

.Resources that are automatically backuped up and restored for Observability
|====
| Resource type | Resource name  

> .5+| ConfigMaps 

Reviewer 2: make sure this is converted, and if not, reconsider how we build this information in the table.

Author: This is how it looks when converted:

<img width="820" height="731" alt="image (24)" src="https://github.com/user-attachments/assets/b6902be9-8f33-4a6d-91ff-213fdf5f6512" />

I don't like how the lines aren't appearing, plus i'm using more lines than needed. Will just do the comma list
@swopebe

Reviewer 2: I don't mind using a method that is accepted widely and will be converted, but usually the best thing to do is keep tables small and clean of extra code.
